runtime/stdlib/crypto.go

```diff
@@ -107,13 +107,16 @@ func newCryptoContractVerifySignatureFunction(signatureVerifier CryptoSignatureV
 			}
 			hashAlgorithm := hashAlgorithmStringValue.Str
 
-			isValid := signatureVerifier.VerifySignature(signature,
+			isValid, err := signatureVerifier.VerifySignature(signature,
 				tag,
 				signedData,
 				publicKey,
 				signatureAlgorithm,
 				hashAlgorithm,
 			)
+			if err != nil {
+				panic(fmt.Errorf("VerifySignature: failed to verify the signature: %w", err))
```

Comment:
Just panic with the error

```suggestion
				panic(err)
```